views/drones/list.hbs

```diff
@@ -1 +1,26 @@
-<h2>Available drones</h2>
\ No newline at end of file
+<h2>Available drones</h2>
+<hr />
+{{#each drones}}
+
+<div>
+    
+    <h3> {{name}} </h3>
+    <p> {{propellers}} </p>
+    <p> {{speed}} </p>
```

Comment:
Be careful here! You're not displaying the `maxSpeed` because you passed it as `speed`.